bin/cli

```diff
@@ -166,7 +171,9 @@ async function runCreate () {
 
   await npmInstall(installDirectory, [kitDependency, 'govuk-frontend'])
 
-  await spawn('npx', ['govuk-prototype-kit', 'init', '--', installDirectory], {
+  const additionalArgs = Object.keys(argv.options).map(name => `--${name}="${argv.options[name]}"`)
+
+  await spawn('npx', ['govuk-prototype-kit', 'init', '--', ...additionalArgs, installDirectory], {
```

Comment:
`additionalArgs` should go at the end, otherwise there may be forwards compatibility issues when the `create` command for a newer version of the package hands over to `init` for an older version.

```suggestion
  await spawn('npx', ['govuk-prototype-kit', 'init', '--', installDirectory, ...additionalArgs], {
```

Reply:
Alternatively, passing the options via environment variables might be cleaner.

Reply:
I like passing on all the arguments because it makes it easier to add other options for that second stage without having to wire them in.

I've switched the order.